OpenSourcePolitics/decidim-tou
# Fix/bump friendly signup

#### :tophat: Description
Bump *friendly_signup: with button disabled when the user enters the registration code.
*This module simply substitutes some pages to ease up the registration process in Decidim.

#### :pushpin: Related Issues
*Link your PR to an issue*
- Related to #?
- Fixes #?
- [Notion card](https://www.notion.so/opensourcepolitics/Decidim-app-Toulouse-Bump-FSU-c98a83902e6744b2a3039fa721d61598?pvs=4)

#### Testing
1. Go to sign up 
2. Enter the registration code received by mail (The 'verify' button has become disabled) 
3. It should automatically display your homepage

Example:
* Log in as admin
* Access Backoffice
* Go to organization settings
* See ...

#### Tasks

### :camera: Screenshots
*Please add screenshots of the changes you're proposing if related to the UI*
